common/source/docs/common-imu-notch-filtering.rst

```diff
@@ -52,10 +52,10 @@ Notch Filter Control Types
 Key to the dynamic notch filter operation is control of its center frequency. There are five methods that can be used for doing this:
 
 #. :ref:`INS_HNTCH_MODE <INS_HNTCH_MODE>` = 0. **Static center frequency**. Dynamic notch frequency control is disabled. The center frequency is fixed and is static. Often used in Traditional Helicopters with external governors for rotor speed, either incorporated in the ESC or separate for ICE motors.
-#. :ref:`INS_HNTCH_MODE <INS_HNTCH_MODE>` = 1. (Default) **Throttle position based**, where the frequency at hover throttle is determined by analysis of logs, and then variation of throttle position above this is used to track the increase in noise frequency. Note that the throttle reference only applies to VTOL motors in QuadPlanes, not forward motors, and will not be effective in fixed wing only flight modes. See :ref:`throttle-based<common-imu-notch-filtering-throttle-based-setup>` for further setup details.
+#. :ref:`INS_HNTCH_MODE <INS_HNTCH_MODE>` = 1. (Default) **Throttle position based**, where the frequency at hover throttle is determined by analysis of logs, and then variation of throttle position above this is used to track the increase in noise frequency. See :ref:`throttle-based<common-imu-notch-filtering-throttle-based-setup>` for further setup details.
```

Comment:
You could add that the throttle-based parameters can also be determined by running an in-flight FFT

Reply:
if your autopilot has the ability to do FFT, why would you do throttle based?

Reply:
Throttle based is very fast - FFT has some phase lag that increases with accuracy. Leonard and I actually discussed making a new mode where the FFT would slowly update the throttle parameters for accuracy but the notch itself would operate using throttle notch

Reply:
done